release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -492,6 +492,45 @@ You can now control the amount of information that is logged to the node audit l
 
 See xref:../nodes/nodes/nodes-nodes-audit-config.adoc#nodes-nodes-audit-config[Configuring the node audit log policy] for more information.
 
+[id="ocp-4-6-logging"]
+=== Cluster logging
+
+[discrete]
+[id="ocp-4-6-logging-api-ga"]
+==== Log Forwarding API is generally available
+
+The xref:../logging/cluster-logging-external.adoc[OpenShift Logging Log Forwarding API] is now generally available. The Log Forwarding API allows you to send container, infrastructure, and audit logs to specific endpoints within and outside your cluster by configuring a custom resource with the endpoints to forward the logs. The Log Forwarding API now supports forwarding to Kafka brokers and supports syslog RFC 3164 and RFC 5424 including TLS. You can also forward application logs from a specific projects to an endpoint. 
```

Comment:
"OpenShift Logging Log Forwarding API" is wordy and we don't use that terminology in the main docs. I would keep it simple and just use "Log Forwarding API"